report.html

```diff
@@ -53,15 +53,21 @@
 
 <script>
   function formSubmitted(f) {
+<<<<<<< HEAD
+=======
```

Comment:
These are merge conflicts. You need to delete the lines that begin with `<<<<`, `====` and `>>>>`